message_test.go

```diff
@@ -21,6 +21,12 @@ func TestClient_PinMessage(t *testing.T) {
 	messageResp, err := resp1.Channel.SendMessage(context.Background(), msg, userB.ID)
 	require.NoError(t, err)
 
+	msgWithOptions := &Message{Text: "test message"}
+	_, err = resp1.Channel.SendMessage(context.Background(), msgWithOptions, userB.ID, func(msg *messageRequest) {
```

Comment:
Can we check that the message is correcty quoted in the response?

Reply:
✅